src/parsers/common.jl

```diff
@@ -123,18 +123,48 @@ function convert_units!(
     value::Float64,
     unit_conversion::NamedTuple{(:From, :To), Tuple{String, String}},
 )
-    if unit_conversion.From == "degree" && unit_conversion.To == "radian"
+    if uppercase(unit_conversion.From) == uppercase("degree") &&
```

Comment:
Would it be better in this case to use `normalize` https://docs.julialang.org/en/v1/stdlib/Unicode/#Unicode.normalize